### PR TITLE
New Court Physician Room, keeps armory expansion, and TONs of map fixes and adjustments V2

### PR DESCRIPTION
**Pull request by TheManWithNoHands**
## About The Pull Request
I did so much, but thanks to merge conflicts its best if i try to fix a lot of things in one go. Fuck...why does shit keep breacking..

Third instance of me trying to make this work since the first try of https://github.com/Blackstone-SS13/BLACKSTONE/pull/983
* Adds a new Court phys room. Replacing the old, unused private liberary of the king. Also it has magictower are becuse that gives it nice music. I make a new one just for it later, shush.
* Makes the keep armory a bit bigger, replacing a lot of iron weapons with steel, adding spare keys, heavy leather gloves, chains, bows and xbows, quivers for each and some empty ones.
* Gives the archives some ligth.
* Swaps the vaults chest with gold ones.
* Replaced the deadbolt doors at the butcher with normal ones. As Butcher players requested.
* Map fixes such as: Doubt gravehols, empty tiles, floating paths and more

*You hear a Zad in the distance*

## Why It's Good For The Game
https://imgur.com/a/flH1k69

The Court Physician is a Nobel and should be treated as such. They are employed by the king to keep him and his court alive. Giving them a room to work in is the least they should have.

"Recent" Pr upgraded the keeps guard gear by a lot, giving them much better gear. So having their armory reflect that is more than fine. Also, this gives the keep chains, which it didn't have for some reason. It is still kind of on the same level of amount of stuff as the bog armory so shh.

Map fixes good.

### CODE REVIEW

File: _maps/map_files/blackstone/blackstone.dmm [hunk 1]
```diff
@@ -2234,7 +2284,7 @@
 "qYw" = (/obj/structure/rack/rogue,/obj/item/rogueweapon/spear,/turf/open/floor/rogue/ruinedwood,/area/rogue/indoors)
 "qYF" = (/obj/structure/fluff/dryingrack,/obj/item/needle/thorn,/turf/open/floor/rogue/cobblerock,/area/rogue/outdoors/rtfield)
 "qYJ" = (/turf/open/floor/rogue/carpet,/area/rogue/indoors/town/tavern)
-"qZw" = (/obj/structure/closet/crate/drawer{pixel_y = 16},/obj/item/roguekey/butcher,/obj/item/roguekey/butcher,/turf/open/floor/rogue/twig,/area/rogue/indoors)
+"qZw" = (/obj/structure/closet/crate/drawer{pixel_y = 16},/obj/item/roguekey/butcher,/obj/item/roguekey/butcher{lockid = s},/turf/open/floor/rogue/twig,/area/rogue/indoors)
```
Comment (JTGSZ) [inline _maps/map_files/blackstone/blackstone.dmm:2287]: ye that lockid = s needs to be a lockid = "s"

Comment (TheManWithNoHands) [reply _maps/map_files/blackstone/blackstone.dmm:2287]: thank you so much

**Comment by Real-MAGNUM** [comment]
how many crossbows in storage?

**Comment by Real-MAGNUM** [comment]
looks good to me

**Comment by ophaq** [comment]
I am surprised you didn't notice the old butcher's basement residing deep underground in a random field where it used to be pre-bogtown and remove it. I don't mind as it is a funny spot to dig to for a free house.

**Comment by TheManWithNoHands** [comment]
> I am surprised you didn't notice the old butcher's basement residing deep underground in a random field where it used to be pre-bogtown and remove it. I don't mind as it is a funny spot to dig to for a free house.

Mainly fixed stuff i people pointed out or i noticed at random. But will fix next map Pr i do

**Pull request merged**